packages/apps/shopify-app-react-router/src/server/__test-helpers/test-config.ts

```diff
@@ -7,17 +7,17 @@ import {testConfig as testConfigImport} from '../test-helpers/test-config';
 
 /*
  * This object mandates that all existing future flags be activated for tests. If a new flag is added, this object must
- * be updated to include it, which will also cause all tests to use the new behaviour by default (and likely fail).
+ * be updated to include it, which will also cause all tests to use the new behavior by default (and likely fail).
```

Comment:
🇬🇧 🥲

Reply:
Lol, I actually hd no idea there was different spellings.  Will revert, don't want to start something!

Reply:
Actually, seems like we use a mix.  Will leave